### PR TITLE
Feature/second tmcr selection

### PR DESCRIPTION
**Pull request by RobertPorterfield**
Do the Table2 PR first as that'll help split out what's specific to this update vs that update, and this one is a bit more involved.

Basically added the logic to handle adding a second TMCR to the wizard (can do two at the same time).
Will likely need another update in the future as they identify duplicates between the two TMCRs

### CODE REVIEW

**Review by GregoryForrest — APPROVED** [review]
Reviewed, added some comments for future considerations.  Approved.

File: src/Steps/Steps.tsx [hunk 1]
```diff
@@ -2,11 +2,14 @@ import { Preface } from "./Preface";
...
 import { TOProgramReqs } from "./TOProgramReqs";
 import { TMSSReqs } from "./TMSSReqs";
+import { Table2 } from "./Table2";
 import { FinalStep } from "./FinalStep";
 
 export const TMCRStepNames = ["Home", "TO Info", "TO Program Requirements", "TMSS Requirements", "IETM Functionaly Requirements", "Mandatory Requirements", "Configurable Requirements", "Delivery Requirements", "Spec/Std Interface Records (SIRS)", "Attachment 1", "Attachment 2", "Review/Print"];
 export const TMCRFinalStep = TMCRStepNames.length-1;
-const WizardSteps = [<Preface />, <TOInfo />, <TOProgramReqs />, <TMSSReqs />, , , , , , , , <FinalStep />];
+// TODO: remove ts-ignore after all steps have been created
+// @ts-ignore
```
[the remark this answers is not in the document]
Comment (RobertPorterfield) [reply src/Steps/Steps.tsx:11]: lol, you are correct! I'll fix that next go round

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [Unreleased]
 - (Keep your changes here until you have a release version)
 
+## [0.1.3]
+### Added
+- Add second TMCR logic
```
[the remark this answers is not in the document]
Comment (RobertPorterfield) [reply CHANGELOG.md:13]: Reset button (on every page) should clear (well, _reset_) everything as needed after I push the update out. I'll be sure to mention it tomorrow. Still only product owners looking at it right now

File: src/stateManagement/reducer.ts [hunk 3]
```diff
@@ -1,33 +1,53 @@
...
+        wizardOptions.push({...defaultOptions});
+        return {
+          ...state,
+          wizardOptions,
+          wizardStep: 1,
+          tmcrIndex: 1
+        }
+      }
       return state;
+    case 'LOAD_TMCR':
```
[the remark this answers is not in the document]
Comment (RobertPorterfield) [reply src/stateManagement/reducer.ts:45]: Good call! I'll look into adding a jsdoc style comment

**Pull request merged**